bcda/client/bluebutton.go

```diff
@@ -96,6 +100,13 @@ func (bbc *BlueButtonClient) GetPatientData(patientID, jobID string) (string, er
 	return bbc.getData(blueButtonBasePath+"/Patient/", params, "")
 }
 
+func (bbc *BlueButtonClient) GetBlueButtonIdentifier(hashedHICN string) (string, error) {
```

Comment:
I'm wondering if we should put the hashing functionality inside this function, so callers don't have to remember to do it?  So it would

Reply:
I kept it out so that if we needed to hash something else we had that option available to us.